src/main/scala/com/ing/wbaa/gargoyle/sts/oauth/KeycloakTokenVerifier.scala

```diff
@@ -0,0 +1,46 @@
+package com.ing.wbaa.gargoyle.sts.oauth
+
+import com.ing.wbaa.gargoyle.sts.config.GargoyleKeycloakSettings
+import org.keycloak.RSATokenVerifier
+import org.keycloak.adapters.KeycloakDeploymentBuilder
+import org.keycloak.representations.adapters.config.AdapterConfig
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+trait KeycloakTokenVerifier {
```

Comment:
Maybe it would be nicer to test this trait separately instead of solely in the global integration test? Maybe a separate it-test for this and then only specify a valid token in the other integration tests. That way it might be a bit more clear where what is tested

Reply:
done